packages/react-devtools-shell/now.json

```diff
@@ -1,5 +0,0 @@
-{
-  "name": "react-devtools-experimental",
```

Comment:
what was this file for?

Reply:
I used to deploy this demo to Now (Vercel) but we don't anymore.